content/en/apps/examples/phc-covid.md

```diff
@@ -44,4 +44,6 @@ Primary health care plays a vital role both in slowing the spread of COVID-19 an
 
 This demo illustrates how a CHT workflow that currently supports PHC can be easily adapted to reflect a COVID-19 health program response. Remote support allows for the integration of new care protocols that follow “no touch” or “visual only” reviews, transition to call/text-first protocols, and embedding COVID-19 symptom screening within PHC workflows.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/ic_sBf4KlYQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
\ No newline at end of file
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ic_sBf4KlYQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+More background information can be found in this [summary deck](https://docs.google.com/presentation/d/1DBbEc2R6-TwgD4pp_LjTmgGXAUzZzeVrYlrDzFdWqSI/edit#slide=id.g7f8f8792f7_0_0).
```

Comment:
Not necessary to accept, but I usually remove the slide id, just in case things move around. 
```suggestion
More background information can be found in this [summary deck](https://docs.google.com/presentation/d/1DBbEc2R6-TwgD4pp_LjTmgGXAUzZzeVrYlrDzFdWqSI).
```